pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -131,11 +135,16 @@ func (c *APIReconciler) reconcile(ctx context.Context, apiExport *apisv1alpha1.A
 			continue
 		}
 
+		logger = logger.WithValues("identity", pc.IdentityHash)
+
+		logger.V(4).Info("getting APIExports by identity")
```

Comment:
Is project policy on logging that `v=4` is effectively "anything goes"? Just notice a lot of messages being propagated throughout here 👍🏻

Reply:
We can lower to 5 or 6 later, or remove them entirely, but this was critical in determining what was going on.

Reply:
ack